{{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/model/prediction.py

```diff
@@ -83,30 +83,5 @@ def cross_entropy(self, examples):
             self.stack_class_indices(examples),
         )
 
-    def cpu(self):
```

Comment:
Why do we remove this and `detach`?

Reply:
They are very ugly and they are not used anymore